packages/coordinator/src/screens/AuthScreens/ResetPasswordScreen.tsx

```diff
@@ -1,33 +1,75 @@
 import { useState } from "react";
 import styles from "./AuthScreens.module.scss";
 import Input from "../../components/Input";
-import Button from "../../components/Button";
+import Button, { ButtonVariant } from "../../components/Button";
 import logoImage from "./../../assets/blood-bank-zichron-logo.svg";
+import popupImage from "../../assets/popup_resetpass.png";
 import chevronSvg from "./../../assets/icons/chevron-right-small.svg";
 import { useNavigate } from "react-router-dom";
+import { Popup } from "../../components/Popup/Popup";
 
 export interface ResetPasswordScreenProps {
   onResetPassword: (
     email: string,
     emailError: (error: string) => void
-  ) => Promise<void>;
+  ) => Promise<boolean>;
 }
 
 export default function ResetPasswordScreen(props: ResetPasswordScreenProps) {
   const [email, setEmail] = useState("");
   const [emailError, setEmailError] = useState("");
   const [isLoading, setIsLoading] = useState(false);
 
+  const [popupOpen, setPopupOpen] = useState<boolean>(false);
+  const [popupError, setPopupError] = useState<string | undefined>(undefined);
+
   const navigate = useNavigate();
 
   const resetPassword = async () => {
     setIsLoading(true);
-    await props.onResetPassword(email, setEmailError);
+    const res = await props.onResetPassword(email, setEmailError);
```

Comment:
Why do you need the `res`?

If this fails, it will call setEmailError. What do you use Res for?

If you want to know whether this succeeded, you can check if this threw.

Reply:
done